samples/dictionary.sample.yml

```diff
@@ -0,0 +1,763 @@
+# Key to Entry Fields
+#
+# Each object represents a *dictionary entry*.
+#
+# category:                                         # The lexical category for the entry. This is displayed to the end user as-is, but it is also used by the dictionary app for ...? For Plains Cree, the value of this field is whatever is in Arok's `\ps` field in the Toolbox file. For now, each entry may only have one lexical category.
+# FST:                                              # Object containing information for the FST.
+#   lemma:                                          # The lemma used by the FST, in the proto-orthography.
+#   stem:                                           # The stem used by the FST, in the proto-orthography.
+# head:                                             # The head(word|morpheme|phrase) displayed to the user, in the proto-orthography. Will get transliterated to other orthographies in the dictionary app depending on which orthography the user selects.
+# key:                                              # A unique key for this entry. Includes a trailing homograph number if the form is a homograph.
+# paradigm: <static> | <dynamic> | "unknown" | null # Indicates the paradigm used by this entry. null = non-inflecting / don't show a paradigm. "unknown" indicates that additional work is needed by linguists---don't show a paradigm (or maybe show "unknown paradigm" or something).
+# POS: N | V | Part | Pro | (others?)               # The general word class / part of speech. Displayed to the user as-is. Used by the dictionary app for emojis and ...?
```

Comment:
Is the logic here the following?:

1. if there is a named static or dynamic paradigm and an FST/lemma for an entry, so that the FST/lemma is equal to the entry head, then a paradigm can be (attempted to be) shown using the FST/lemma and the specified paradigm?

2. If there is no FST/lemma, then there would be no specified paradigm either, and no paradigm will be shown (nor attempted).

But what if:

3. The entry head is not a lemma (baseform), but one can analyze the entry head so that one can retrieve an FST/lemma. Is this lemma retrieved from the FST-analysis what is assigned as the FST/lemma? And then, a paradigm is attempted if the FST/lemma is a head word of some other entry, for which option 1 applies (i.e. that entry has a specified paradigm)?

4. The entry can be assigned an FST/lemma, but no paradigm has been assigned (e.g. Ipc's). In such a case, no paradigm is shown?

As examples:

1. Category 1 concerns entries for nouns and verbs (dynamic) as well as pronouns (static), where the entry head is the same of the FST/lemma retrieved via FST analysis.

2. Category 2 concerns entries for morphemes for which there are no FST analyses, nor any assigned paradigm.

3. Category 3 concerns entries for inflected wordforms for which there is an FST analysis, but the provided lemma is not the same as the entry head. The paradigm should be dependent on that assigned to another entry with a head that corresponds to the FST/lemma.

4. Category 4 concerns entries for non-inflecting word-classes for which one can assign an FST analysis and extract a lemma, but there are no meaningful paradigms. E.g. Indeclinable particles (perhaps also multi-word phrases).

Further questions/comments:
a. In the case of the FST/lemma being ambiguous as to what entry it would match, perhaps we should also specify the unique key for the intended entry?
b. In my view, I think we'd want to include here besides the lemma also the unambiguous FST analysis.